base/workflow/man/run.write.configs.Rd

```diff
@@ -2,39 +2,34 @@
 % Please edit documentation in R/run.write.configs.R
 \name{run.write.configs}
 \alias{run.write.configs}
-\title{Write model-specific run scripts and configuration files}
+\title{Run model specific write configuration functions}
 \usage{
 run.write.configs(settings, write = TRUE,
   ens.sample.method = "uniform", posterior.files = rep(NA,
   length(settings$pfts)), overwrite = TRUE)
 }
 \arguments{
-\item{settings}{a PEcAn settings list}
-
-\item{write}{should the runs be written to the database?}
+\item{write}{should the runs be written to the database}
 
 \item{ens.sample.method}{how to sample the ensemble members('halton' sequence or 'uniform' random)}
 
 \item{posterior.files}{Filenames for posteriors for drawing samples for ensemble and sensitivity
-analysis (e.g. post.distns.Rdata, or prior.distns.Rdata)}
+analysis (e.g. post.distns.Rdata, or prior.distns.Rdata). Defaults to NA, in which case the 
+most recent posterior or prior (in that order) for the workflow is used. Should be a vector, 
+with one entry for each PFT. File name only; PFT outdirs will be appended (this forces use of only
+files within this workflow, to avoid confusion).}
 
-\item{overwrite}{logical: Replace output files that already exist?}
+\item{model}{the ecosystem model to generate the configuration files for}
 }
 \value{
 an updated settings list, which includes ensemble IDs for SA and ensemble analysis
 }
 \description{
-Generates run scripts and configuration files for all analyses specified
-in the provided settings. Most of the heavy lifting is done by the
-\code{write.config.*} function for your specific ecosystem model
-(e.g. write.config.ED2, write.config.SIPNET).
+Main driver function to call the ecosystem model specific (e.g. ED, SiPNET) 
+run and configuration file scripts
 }
 \details{
-The default value for \code{posterior.files} is NA, in which case the
-   most recent posterior or prior (in that order) for the workflow is used.
-   When specified, \code{posterior.files} should be a vector of filenames with one entry for each PFT.
-   Specify filenames with no path; PFT outdirs will be appended. This forces use of only
-   files within this workflow, to avoid confusion.
+DEPRECATED: This function has been moved to the PEcAn.workflow package and will be removed from PEcAn.utils.
```

Comment:
1. Why are there changes to run.write.configs.Rd in a PR that doesn't change run.write.configs.R? 

2.  This is the man page for `PEcAn.workflow::run.write.configs`, which is the version that isn't deprecated, so it should definitely NOT contain the deprecation notice from `PEcAn.utils::run.write.configs`.

Reply:
Ugh.  That's from a mix of bad version controlling and the switch to roxygen2 6.1.0.  After I ran `make` after updating to 6.1.0, there were about a large number of .Rd files to commit.  Additionally, I'd been using an intermediate version of run.write.configs in for code that was being developed concurrently.  During part of that process, run.write.configs was moved to PEcAn.utils.  

In any case, it's a mistake, so I'll get that fixed.